example/scalalib/linting/2-contrib-scoverage/build.mill

```diff
@@ -50,6 +53,13 @@ scoverage.xmlCoberturaReport
 ...
 scoverage.xmlReport
 ...
+scoverage.validateCoverageMinimums
+
+> ./mill scoverage.validateCoverageMinimums
+...
+Coverage Minimums Met. Minimums set: {Statements: 10.00% Branches: 90.00%}
+Statement coverage.: 10.53%
+Branch coverage....: 100.00%
```

Comment:
By prepending this with "error:", we can assert that the `validateCoverageMinimums` task indeed fails.

Reply:
Looks like your change fixed this. But now we're seeing a bloken java android tests.

>  [1] selective.run 1 tasks failed
[8189] example.androidlib.java[1-hello-world].local.daemon.testForked 1 tests failed: 
  mill.testkit.UtestExampleTestSuite mill.testkit.UtestExampleTestSuite.exampleTest
> Error: Process completed with exit code 1.

I think we're good.. I think that's just a flaky test.